external/src/main/java/com/redhat/cloud/policies/engine/lightweight/AccountLatestUpdate.java

```diff
@@ -10,9 +10,13 @@
 @Table(name = "account_latest_update")
 public class AccountLatestUpdate {
 
+    // TODO POL-650 shouldn't we use auto generated ids as PK instead of natural keys?
     @Id
```

Comment:
There should only be one entry per account, so account-id as PK sounds correct to me.

Reply:
Ok. A matter of taste maybe. That makes some use cases a bit harder than they could be.

Reply:
Removed